internal/errs/userfacing.go

```diff
@@ -27,19 +28,24 @@ func (e *userFacingError) ErrorTips() []string {
 	return e.tips
 }
 
+func (e *userFacingError) InputError() bool {
```

Comment:
If I recall correctly from my previous PR we wanted to avoid user facing errors from colliding with existing error implementations. This is an implementation of the `ErrorInput` interface from the locale package.

Reply:
This isn't a collision. The problem I ran into is wanting to communicate an input failure on the user facing error side. So when I asserted the type I could set the user facing error, but also assert that it is due to user input.

If we use WrapInputError or something like that we add even more errors to the stack, and the API would be pretty awkward. This solves both those problems, you can now set a user facing error as an input error with `NewUserFacing(msg, SetInput(true))`